jobs/scheduled/monitor_event_dates.rb

```diff
@@ -39,14 +39,21 @@ def finish(event_date)
 
       return if event_date.event.recurrence.blank?
       event_date.event.set_next_date
+      event_date.event.set_topic_bump
     end
 
     def due_reminders(event_date)
       return [] if event_date.event.reminders.blank?
-      event_date.event.reminders.split(",").map do |reminder|
-        value, unit = reminder.split('.')
+      event_date.event.reminders.split(',').map do |reminder|
 
-        next if !validate_reminder_unit(unit)
+        reminder_array = reminder.split('.')
+        if reminder_array.length() === 3
+          type, value, unit = reminder_array
+        else
+          value, unit = reminder_array
+        end
```

Comment:
We're doing this on multiple places and it feels a little flimsy. It might be good to move this logic to a helper in the model and call that. Or... add a migration and avoid this altogether. 

Or... since we can only set one bump per topic, the data for that could be on its own column? So in UI, we can still show the dropdown, but in backend we store it separate, i.e.: 

```
reminders: "15.minutes,-30.minutes",
topicBump: "10.minutes",
```

I'm mostly thinking out loud here, happy to hear your thoughts.

Reply:
> Or... since we can only set one bump per topic, the data for that could be on its own column?

It would be similar to how I made it previously. I wanted to be as consistent as possible across UI, raw, and models. At least for me, it's clearer that way. I think adding a migration for that would be good, though.

Reply:
Yeah, I think you're right, consistency is best. Let's look into a migration, that's probably a good next step.